client/spec/authoring_spec.js

```diff
@@ -178,4 +179,12 @@ describe('authoring', function() {
         expect(authoring.getHistoryItems().count()).toBe(1);
         expect(authoring.getHistoryItem(0).getText()).toMatch(/Published by.*/);
     });
+
+    it('allows to create a new empty package', function () {
+        monitoring.openMonitoring();
+        browser.sleep(500);
```

Comment:
(while we are waiting for the build to start...)

I have noticed this practice of using `browser.sleep()` in several tests, but I'm not sure this is the right approach. In general, shouldn't we use protractor's promises returned by triggered actions (`click()`, `setLocation()`, etc.) instead to wait for these actions to complete? Or am I missing something?

Thanks.